src/utils.rs

```diff
@@ -113,10 +113,7 @@ fn get_corrected_path(path: PathBuf) -> PathBuf {
     }
 }
 
-fn get_cabinet_files<'a>(
-    cab: &'a Cabinet<Cursor<&Vec<u8>>>,
-    path: PathBuf,
-) -> Option<(String, usize)> {
+fn get_cabinet_files(cab: &Cabinet<Cursor<&Vec<u8>>>, path: PathBuf) -> Option<(String, usize)> {
```

Comment:
clippy says the `<'a>` is an unnecessary lifetime and can be elided over, so eliding we will go!